Doc/c-api/unicode.rst

```diff
@@ -1396,10 +1396,12 @@ They all return ``NULL`` or ``-1`` if an exception occurs.
    :c:func:`PyErr_Occurred` to check for errors.
 
 
-.. c:function:: int PyUnicode_EqualToString(PyObject *unicode, const char *string)
+.. c:function:: int PyUnicode_EqualToUTF8(PyObject *unicode, const char *string)
 
    Compare a Unicode object with a UTF-8 encoded C string and return true
    if they are equal and false otherwise.
```

Comment:
It looks to me, that "return true" is more often used than "return non-zero". In this case it is more accurate, because it always returns 1, not other non-zero value. Perhaps other functions which return non-zero was a macro that returned not 1 (something like `(arg->flags & FLAG)`)?